packages/widget/src/components/address-input/styles.ts

```diff
@@ -0,0 +1,36 @@
+import { css } from 'lit';
+
+export const styles = css`
+  .addressInputContainer {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 8px;
+  }
+
+  .inputAddressContainer {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 91px;
```

Comment:
not a big fan of 'hardcoded' values for height/width as they make style harder to maintain. we can discuss other approaches on slack :)

Reply:
I think we should create an issue addressing styling overall over the widget, so we do not loose the scope of this PR. 100% agree